tests/src/main/scala/tests.scala

```diff
@@ -176,5 +176,13 @@ object Tests extends TestApp {
     test("serialize a value class") {
       Show.gen[Length].show(new Length(100))
     }.assert(_ == "100")
+
+    test("show a case class with repeated param") {
```

Comment:
Could you include a second test which handles a repeated parameter which is also a Magnolia-derived type, please? Not likely to be a problem, but an additional test would be useful.

Reply:
I'm sorry I don't follow. What do you mean by Magnolia-derived type?

Reply:
I mean a repeated type which needs Magnolia to derive the typeclass, i.e. not a `String` (`Seq[String]`), but a `Foo` (`Seq[Foo]`) for some case-class type, `Foo`.

Reply:
Oh I see, will do :+1: